app/src/core/design/components/userComponents/Button/Button.js

```diff
@@ -13,7 +13,6 @@ export const Button = ({ props, parentStyle, style, ...rest }) => {
     if (parentStyleCopy.backgroundImage !== "") {
         parentStyleCopy.backgroundImage = "url(" + parentStyleCopy.backgroundImage + ")";
     }
-
```

Comment:
create a method here: getBorderStyles(style) and returns border style object. do the checks in here and return that object.
also add a comment to this method on why you are writing this fucntion